tests/hazmat/primitives/test_ciphers.py

```diff
@@ -255,7 +255,12 @@ def test_update_into_buffer_too_small_gcm(self, backend):
     sys.platform not in {"linux", "darwin"}, reason="mmap required"
 )
 def test_update_auto_chunking():
-    large_data = mmap.mmap(-1, 2**29 + 2**20, prot=mmap.PROT_READ)
+    # Silencing mypy warning on Windows, even though mmap doesn't exist. See:
+    # https://mypy.readthedocs.io/en/stable/common_issues.html#version-and-platform-checks
+    if sys.platform == "win32":
```

Comment:
Unfortunately this will produce coverage issues, since it's never reached.

I wonder if an `assert sys.platform in {"linux", "darwin"}` would work?

Reply:
I've seen it in the documentation, and tried it, as well as `assert sys.platform != "win32"`, without any success. it seems it only works to silence the whole file 😞